client/my-sites/marketing/connections/service.jsx

```diff
@@ -112,12 +115,24 @@ export class SharingService extends Component {
 			'must-disconnect' === connectionStatus
 		) {
 			this.removeConnection();
+			this.props.recordTracksEvent( 'calypso_connections_disconnect_button_click', {
+				service: this.props.service.ID,
+				path,
+			} );
 			this.props.recordGoogleEvent( 'Sharing', 'Clicked Disconnect Button', this.props.service.ID );
```

Comment:
In that case it would mess up the existing analytics, and since we won't need Google Analytics after switching to Tracks I think it's fine to just leave it.